src/data-pipeline/spark-etl/src/main/java/software/aws/solution/clickstream/UserPropertiesConverter.java

```diff
@@ -69,6 +70,35 @@ private static UDF1<String, Row> convertUserLtv() {
     @NotNull
     private static GenericRow[] getUserPropertiesGenericRows(final String value) throws JsonProcessingException {
         ObjectMapper objectMapper = new ObjectMapper();
+
+    /*
```

Comment:
add a detail info about this sample data

Reply:
move the comments line to java method doc comments and add description.